Java/src/main/java/planes/Plane.java

```diff
@@ -15,30 +15,25 @@ public Plane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapac
         this.maxLoadCapacity = maxLoadCapacity;
     }
 
-    public String getModel() {
-        return model;
-    }
-
-    public int getMS() {
+    public int getMaxSpeed() {
         return maxSpeed;
     }
 
-    public int Get_Max_Flight_Distance() {
+    public int getMaxFlightDistance() {
         return maxFlightDistance;
     }
 
-    public int getMinLoadCapacity() {
-        int result = this.maxLoadCapacity;
-        return result;
+    public int getMaxLoadCapacity() {
+        return maxLoadCapacity;
     }
 
     @Override
     public String toString() {
-        return "Plane{" +
+        return "\nPlane{\n" +
```

Comment:
"\nPlane" do you need the empty line above Plane?

Reply:
Not related to task. 
Done this for better visibility for me. Without /n newline symbol all text was in one line and not readable